src/middlewared/middlewared/alembic/versions/12.0/2020-02-10_09-47_ssh_weak_ciphers.py

```diff
@@ -0,0 +1,61 @@
+"""SSH weak ciphers
+
+Revision ID: 06bfbd354deb
+Revises: 4abbf75347b2
+Create Date: 2020-02-10 09:47:12.017225+00:00
+
+"""
+import json
+import re
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '06bfbd354deb'
+down_revision = '4abbf75347b2'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    with op.batch_alter_table('services_ssh', schema=None) as batch_op:
+        batch_op.add_column(sa.Column('ssh_weak_ciphers', sa.TEXT(), nullable=True))
```

Comment:
Curious why create with nullable True to only make it False later? Why not create it with `[]` as default? Not possible?

Reply:
Possible, but it will have the default value stored in the schema which is not always desirable. I guess this is not the case for NAS and it's just the matter of habit, thank you for reminding.